tools/star-fusion/Dockerfile

```diff
@@ -6,6 +6,7 @@ LABEL authors="rickard.hammaren@scilifelab.se, phil.ewels@scilifelab.se, martin.
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 ENV PATH /opt/conda/envs/star-fusion/bin:$PATH
+ENV TRINITY_HOME /opt/conda/opt/trinity-2.6.6
```

Comment:
Do you need this?

Reply:
The older version of STAR-Fusion required it. Maybe in the new version, I will drop it.

Reply:
Okay 👌